mercat/cli/common/src/harness.rs

```diff
@@ -345,6 +455,12 @@ impl Transfer {
         let tx_id = self.tx_id;
         let reject = !self.mediator_approves;
         let cheat = self.mediator.cheater;
+        let auditors: Vec<String> = self
+            .auditors
+            .clone()
+            .into_iter()
+            .map(|auditor| auditor.name)
+            .collect();
```

Comment:
If you only need the `auditor.name`, only clone this member. 
Pls note that `into_iter` has been transformed into just `iter`, to use references (you don't need to consume it).
```suggestion
        let auditors = self.auditors.iter().map(|auditor| auditor.name.clone()).collect::<Vec<_>>();
```